app/views/admin/editions/new.html.erb

```diff
@@ -9,6 +9,14 @@
       items: secondary_navigation_tabs_items(@edition, request.path),
     } %>
 
+    <% if @edition.new_content_warning %>
+      <div class="format-advice">
```

Comment:
What impact does this class/div have? I can't see that it's used in a stylesheet from this or another alphagov repo, and it doesn't seem to be used as a test selector here

Although this is a kind of format advice, so far it seems to be used for the content that starts "Use this format for:"

Reply:
As far as I can see, the class is redundant.  But I've chosen to keep it there as it's consistent with other similar pieces of advice, which appear visually in the same place on the form.